.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # pin@v3.5.0
```

Comment:
Why do you pin here but unpinned some other dependencies? We should keep this consistent.